src/services/completions.ts

```diff
@@ -727,6 +727,7 @@ namespace ts.Completions {
     export const enum CompletionKind {
         ObjectPropertyDeclaration,
         Global,
+        Local,

```

Comment:
This name is very confusing.. May be name this ExportSymbols or something ?

Reply:
They’re not symbols that _are_ exported, they’re symbols that _could_ be exported, which are restricted to local declarations of the module... `ModuleLocalDeclaration`?

I’m actually not sure if a new kind is even needed—it seems that it would work the same if set to `None`. I’m also not sure what LS consumers use the `isGlobalCompletion` property for in the first place.

Reply:
Yeah I as you said None is good and doesn't need additional property here.. I think we use this to determine if completion should be memberCompletion or not.. I don't remember exactly but isGlobalCompletion and isMemberCompletion are age old properties